src/CSFeeDistributor.sol

```diff
@@ -121,6 +121,8 @@ contract CSFeeDistributor is
             );
         }
 
+        emit LastDistributed(distributed);
```

Comment:
```suggestion
        emit FeeDistributed(distributed);
```

Reply:
We already have an event with this name.

Reply:
`DistributionSharesUpdated`? similar to others here
also do we need to emit this with zero amount?

Reply:
> `DistributionSharesUpdated`? similar to others here

It would make sense if it was to update the value and to log the new one, but it's just an increment. Alternatively, we can emit both the old and new values and the difference will show the distributed value, but I'm not sure it makes a lot of sense.

> also do we need to emit this with zero amount?

I think so. We will have a new log anyway and it makes sense to log the distributed value, even if it's zero, to make it explicit and consistent.